src/common/components/clusterConfiguration/DiscoveryTroubleshootingModal.tsx

```diff
@@ -13,19 +13,21 @@ import { PrismCode } from '../ui';
 
 export type HostsNotShowingLinkProps = {
   setDiscoveryHintModalOpen: (isOpen: boolean) => void;
+  isSNO?: boolean;
 };
 
 type DiscoveryTroubleshootingModalProps = {
   setDiscoveryHintModalOpen: (isOpen: boolean) => void;
-  isOpen: boolean;
+  isOpen?: boolean;
```

Comment:
Is there a need isOpen changed to optional?

Reply:
Yes, since I reverted the code from CIM which also usies these components, it would result in Linting / TS errors.

Reply:
Sorry, as discussed this was the wrong parameter to mark as optional. I'll fix this :)